CorsixTH/Lua/entities/humanoids/staff.lua

```diff
@@ -440,6 +439,18 @@ function Staff:goToStaffRoom()
   end
 end
 
+-- Function to set pickup action on staff. Pickup action can be deferred.
+function Staff:setPickup(ui, windowToClose)
+  if not self.pickup then -- check if we already added pickup Action in actions queue
+    self.pickup = true
+    local pickupAction = PickupAction(ui)
+    if windowToClose then -- if we want to close some window after pickup happened
+      pickupAction = pickupAction:setTodoClose(windowToClose)
+    end
+    self:setNextAction(pickupAction, true)
+  end
+end
```

Comment:
To summarise a length discord discussion with @Alberth289346 `windowToClose` and `pickupAction` should be converted to snake case instead.
Will also make sure the Wiki is updated to be clearer.

Reply:
Done  👌